libs/core/src/lib/button/button.component.ts

```diff
@@ -37,14 +37,21 @@ import { applyCssClass } from '@fundamental-ngx/core/utils';
     changeDetection: ChangeDetectionStrategy.OnPush,
     host: {
         '[attr.type]': 'type',
-        '[attr.disabled]': '_disabled || null'
+        '[attr.disabled]': '_disabled || null',
+        '[attr.aria-label]': 'ariaLabel ? ariaLabel : specialButtonType.includes(fdType) ? label !==null ?  label +" ,"+ fdType : glyph !== undefined || glyph !== null ? fdType +","+ glyph.split("-").join(" ")  : title : title'
```

Comment:
Looks to hard to read.

Please use `!= null` for checking for not null and not undefined.

Instead of `a ? a : b` you can use `a || b`.

Please rewrite this expression.

Reply:
@platon-rov this(a ? a : b you can use a || b) doesnt work as expected, when ariaLabel is passed the ariaLabel has to be retained. a || b doesn't do that.